src/Arcus.Templates.WebApi/Program.cs

```diff
@@ -365,7 +365,14 @@ private static void ConfigureApp(IApplicationBuilder app)
             app.UseHttpCorrelation();
 #endif
             app.UseRouting();
-            app.UseRequestTracking(options => options.OmittedRoutes.Add("/"));
+            app.UseRequestTracking(options =>
+            {
+                options.OmittedRoutes.Add("/");
+#if OpenApi
+                options.OmittedRoutes.Add("/api/docs");
+                options.OmittedRoutes.Add("/api/v1/docs.json"); 
+#endif
```

Comment:
This is indeed really helpful !

Reply:
Yeah, thought that this would be good to add 😉 .